src/admin/adminActions.ts

```diff
@@ -10,14 +10,23 @@ export const postEvent: ThunkActionCreator = (event) => async (dispatch) => {
   return new Promise(async (resolve, reject) => {
     try {
       const url = `${Config.API_URL}${Config.routes.events.event}`;
-      await fetchService(url, 'POST', 'json', {
+      const data = await fetchService(url, 'POST', 'json', {
         requiresAuthorization: true,
         payload: JSON.stringify({
           event,
         }),
         onFailCallback: () => dispatch(logoutUser()),
       });
 
+      const formdata = new FormData();
+      formdata.append('image', event.cover);
+
+      const url2 = `${Config.API_URL + Config.routes.events.picture}/${data.event.uuid}`;
```

Comment:
Descriptive variable nitpick:
```suggestion
      const imageUploadRoute = `${Config.API_URL + Config.routes.events.picture}/${data.event.uuid}`;
```